assets/scss/modules/_carousel.scss

```diff
@@ -2,9 +2,193 @@
 /// This is a poster comment.
 /// It will apply annotations to all items from file.
 /// @access private
-/// @author
+/// @author Ornella Cicalello
 /// Classes for the Carousel component
 ////
 
 .dp-library {
+  .dp-carousel {
+    position: relative;
+    padding-bottom: $dp-spaces--lv5;
+
+    &-wrapper {
+      width: 100%;
+      height: auto;
+      border: 2px solid $dp-color-silver;
+      background: $dp-color-ghostwhite;
+      border-radius: 3px;
+      overflow: hidden;
+    }
+
+    &-content {
+      -webkit-display: inline-flex;
+      -moz-display: inline-flex;
+      -ms-display: inline-flex;
+      -o-display: inline-flex;
+      display: inline-flex;
+
+      &.animating {
+        animation: slide-fade 1s;
+      }
+      @keyframes slide-fade {
+        0% {
+          opacity: 0;
+        }
+        100% {
+          opacity: 1;
+        }
+      }
+    }
+
+    &-slide {
+      position: relative;
+      display: none;
+      min-height: 205px;
+      padding: $dp-spaces--lv3;
+      animation: fade-slide 1s;
+
+      @keyframes fade-slide {
+        0% {
+          opacity: 0;
+        }
+        100% {
+          opacity: 1;
+        }
+      }
+
+      @include media-breakpoint('mobile') {
+        min-height: 300px;
+      }
+
+      &:first-child {
+        display: block;
+      }
+
+      span {
+        font-size: 15px;
+        font-weight: 700;
+        color: $dp-color-lightgrey;
+        margin-bottom: $dp-spaces--lv3;
+        display: block;
+      }
+
+      h3 {
+        font-size: 23px;
```

Comment:
is there this size?

Reply:
No I did it like this on the prototype. But I am changing it with a size of the variables